frontend/assets/javascripts/src/modules/form/validation/existEmail.es6

```diff
@@ -0,0 +1,32 @@
+import ajax from 'ajax';
+import debounce from 'lodash/debounce'
+
+export function init() {
+    const emailField = document.getElementById('email');
+    const isUseErrorMessage = document.getElementById('form-field__error-message-email-checker');
+    const checkerURL = '/user/check-existing-email';
+
+    const validateEmail = debounce((event) => {
+        debugger;
+        let prospectiveEmail = emailField.value;
+
+        if(!prospectiveEmail.includes('@') || !prospectiveEmail.includes('.')) {
+            return;
+        }
+
+        ajax({
+            url: checkerURL + '?email=' + emailField.value,
+            method: 'get',
+            success: ({emailInUse}) => {
+                if(emailInUse){
+                    isUseErrorMessage.classList.remove('is-hidden')
+                }else {
+                    isUseErrorMessage.classList.add('is-hidden')
+                }
+            },
+        });
+
+    }, 250);
+
+    emailField.addEventListener('keyup', validateEmail)
```

Comment:
I totally forgot the semicolon in that line. Nevertheless, a little note for the next project, we should probably consider remove `;` from js files. In addition, we should add a JS code validator.